costar_dataset/__init__.py

```diff
@@ -2,4 +2,21 @@
 
 __version__ = '0.4.0'
 __author__ = 'Andrew Hundt <ATHundt@gmail.com>'
-__all__ = []
+
+try:
+    from costar_dataset.block_stacking_reader_torch import CostarBlockStackingDataset
+    from costar_dataset.hypertree_pose_metrics_torch import cart_error, angle_error, grasp_acc_in_bins_batch
+except ImportError:
+    print("Torch not installed.")
```

Comment:
This print statement doesn't seem true... why would it only error on torch and not other missing libraries?